features/steps/steps.go

```diff
@@ -17,13 +17,16 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// OutputFilePath is used for event received by the service.
+var OutputFilePath = "/tmp/dpSearchDataExtractor.txt"
```

Comment:
```suggestion
const outputFilePath = "/tmp/dpSearchDataExtractor.txt"
```
The value is constant hence it should be a constant instead of a var
Secondly, this is only used in the `steps` package so it doesn't need to be exported (with the initial uppercase)